Stripe/PublicHeaders/STPPaymentIntentParams.h

```diff
@@ -47,6 +47,21 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, copy, readwrite) NSString *clientSecret;
 
+/**
+ Provide a supported `STPPaymentMethodParams` object, and Stripe will create a
+ PaymentMethod during PaymentIntent confirmation.
+ 
+ @note alternative to `paymentMethodId`
+ */
+@property (nonatomic, strong, nullable, readwrite) STPPaymentMethodParams *paymentMethodParams;
+
+/**
+ Provide an already created PaymentMethod's id, and it will be used to confirm the PaymentIntent.
+ 
+ @note alternative to `paymentMethodParams`
+ */
+@property (nonatomic, copy, nullable, readwrite) NSString *paymentMethodId;
```

Comment:
I much prefer `Identifier` to `Id` or `ID`. Since we already have `sourceId` on this class I guess `paymentMethodId` makes sense though so just consider this a note for any future/new work :)